src/main/java/com/tasker/server/rest/RecordNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package com.tasker.server.rest;
+
+public class RecordNotFoundException extends RuntimeException{
+
+    public RecordNotFoundException(String message) {
+        super(message);
+    }
+
+    public RecordNotFoundException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public RecordNotFoundException(Throwable cause) {
+        super(cause);
+    }
```

Comment:
Why there are 3

Reply:
I don't know :P, probably no need for others except 1st one.  will change